code/backend/batch/utilities/helpers/EnvHelper.py

```diff
@@ -71,9 +71,19 @@ def __load_config(self, **kwargs) -> None:
             "AZURE_SEARCH_CONVERSATIONS_LOG_INDEX", "conversations"
         )
         # Integrated Vectorization
+        self.AZURE_SEARCH_DATASOURCE_NAME = os.getenv(
+            "AZURE_SEARCH_DATASOURCE_NAME", ""
+        )
+        self.AZURE_SEARCH_INDEXER_NAME = os.getenv("AZURE_SEARCH_INDEXER_NAME", "")
         self.AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION = self.get_env_var_bool(
             "AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION", "False"
         )
+        self.AZURE_SEARCH_IV_MAX_PAGE_LENGTH = os.getenv(
```

Comment:
Could you either make the defaults strings, or wrap the result in `int()`, to avoid typing issues in the future?